source/MissionAction.cpp

```diff
@@ -72,8 +72,18 @@ void MissionAction::Load(const DataNode &node)
 	if(node.Size() >= 3)
 		system = node.Token(2);
 
-	for(const DataNode &child : node)
-		LoadSingle(child);
+	if(node.HasChildren())
+	{
+		auto it = node.begin();
+		if(it->Token(0) == "triggers for failed missions")
```

Comment:
Why can't this just be a part of LoadSingle?
Also, how about `"can trigger after failure"` for the token name? It's still rather lengthy compared to our usual tokens, I think it's a clearer description of the behavior.